utils.go

```diff
@@ -45,12 +45,20 @@ func checkArgs(context *cli.Context, expected, checkType int) error {
 	return nil
 }
 
+func logrusToStderr() bool {
+	l, ok := logrus.StandardLogger().Out.(*os.File)
+	return ok && l.Fd() == os.Stderr.Fd()
```

Comment:
Is this still needed? ISTR old versions of logrus writing to stdout, but they changed that (at least by default) in later versions

Reply:
It is needed because there is a way to redirect the logging to a file, and I did not want to introduce an additional "logging is redirected" flag to he code.

Reply:
Ah; gotcha